plugins/consul/consul.go

```diff
@@ -458,11 +460,12 @@ type serviceListing map[string][]string
 func (sl serviceListing) FindWithTag(t string) serviceListing {
 	results := serviceListing{}
 	for k, v := range sl {
-		if indexof.String(v, t) != -1 {
```

Comment:
And maybe you want to change - https://github.com/turbinelabs/nonstdlib/blob/master/arrays/indexof/indexof.go#L55

from `return ss[i] == target` to `return strings.HasPrefix(ss[i], target)`.

Reply:
we have calling code that expects exact matches there, so probably not.

Reply:
ok, then it's fine the way it is.